src/app/technical-record/adr-details/adr-details-form/store/adrDetails.reducer.spec.ts

```diff
@@ -46,6 +46,15 @@ describe('adrDetailsReducer', () => {
       expect(result.additionalNotesOptions.length).toBe(1);
       expect(result.additionalNotesOptions[0]).toBe('12');
     });
+
```

Comment:
this old adr-details files will be deleted

Reply:
I know what you are saying. We had a chat regarding this some time ago. I added this to improve coverage on the app. This can be removed when you are going to merge the ADR work into develop.